ptcg_cockatrice.xml

```diff
@@ -478,10 +478,10 @@
 			<type>Pokémon - Electric Steel</type>
 			<pt>0/0</pt>
 			<tablerow>2</tablerow>
-			<text>3: Evolve with 2 Magnemite to Magneton
+			<text>3: Evolve with two Magnemite to Magneton
```

Comment:
since you literally can't have 3 magnemite I think we need to rework this whole dude :/

Reply:
Yeah... Maybe replace the X: ability with "1: Create a token that's a copy of Magnemite." Then it or the tokens can still evolve together and give Magneton +3/+3.